litmus-portal/graphql-server/pkg/project/create.go

```diff
@@ -75,11 +75,8 @@ func SendInvitation(ctx context.Context, member model.MemberInput) (*model.Membe
 
 	if invitation == dbSchema.AcceptedInvitation {
 		return nil, errors.New("This user is already a member of this project")
-	} else if invitation == dbSchema.PendingInvitation {
-		err = database.UpdateRole(ctx, member.ProjectID, member.UserName, member.Role, dbSchema.PendingInvitation)
-		return nil, err
-	} else if invitation == dbSchema.DeclinedInvitation {
-		err = database.UpdateRole(ctx, member.ProjectID, member.UserName, member.Role, dbSchema.PendingInvitation)
+	} else if invitation == dbSchema.PendingInvitation || invitation == dbSchema.DeclinedInvitation {
```

Comment:
No need to check these two conditions as if it is not `AcceptedInvitation` will surely be one of the other two

Reply:
If I don't specifiy the invitation status in condition, while sending the invitation for the first time, the user won't get added to the project, function will return before line 83 is executed.